### PR TITLE
Add Unit Test Github Action Workflow 

### PR DESCRIPTION
**Pull request by alexju**
[Short description explaining the high-level reason for the pull request]

## Additions

- Add Unit Test Run for PR and Push on Master

## Removals

-

## Changes

-

## Testing

1. src/troute-config

## Notes

-

## Todos

-

## Checklist

- [x ] PR has an informative and human-readable title
- [x ] Changes are limited to a single goal (no scope creep)
- [x ] Code can be automatically merged (no conflicts)
- [x ] Code follows project standards (link if applicable)
- [x ] Passes all existing automated tests
- [x ] Any _change_ in functionality is tested
- [x ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future todos are captured in comments
- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)
- [x ] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### Target Environment support

- [ ] Windows
- [x ] Linux
- [ ] Browser

### Accessibility

- [ ] Keyboard friendly
- [ ] Screen reader friendly

### Other

- [ ] Is useable without CSS
- [ ] Is useable without JS
- [ ] Flexible from small to large screens
- [ ] No linting errors or warnings
- [ ] JavaScript tests are passing

### CODE REVIEW

**Comment by alexju** [comment]
@taddyb Please review this PR to your development branch.

File: .github/workflows/ubuntu-unit-test.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# This workflow will install Python dependencies, build and run tests on the python versions in the matrix.
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
+
+name: T-route Unit Test
```
Comment (taddyb) [inline .github/workflows/ubuntu-unit-test.yml:4]: Change this to:
`T-Route Pytests`

**Review by taddyb — CHANGES_REQUESTED** [review]
Thanks for the code additions. Lastly, can you change this file name to:
`troute-tests.yml` to ensure the user knows it's for running tests?

**Comment by alexju** [comment]
> Thanks for the code additions. Lastly, can you change this file name to: `troute-tests.yml` to ensure the user knows it's for running tests?

File name changed.

**Comment by alexju** [comment]
@taddyb can you review this pull request when you get a chance?

**Pull request closed without merging**